apl/scalar-functions/conversion-functions/tostring.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: tostring
+description: 'This page explains how to use the tostring function in APL.'
+---
+
+Use the `tostring` function to convert various data types to a string representation. This is helpful when you need to normalize values from different sources into string format for string operations, concatenation, or display purposes.
+
+You typically use `tostring` when working with numeric values, booleans, or other types that need to be converted to strings for string manipulation, formatting, or when combining values in string operations.
```

Comment:
more conversion specifics:
 - ints are formatted as base-10
 - floats are formatted as defined by the g string here: https://pkg.go.dev/strconv#FormatFloat with -1 for precision (this is likely too much details for docs, but just noting it)
 - bools return as "true"/"false", and nil bools return "false"
 - durations return duration formatted strings like 1m20s
 - datetimes return an RFC3339Nano formatted datetime